.github/workflows/publish-arc.yaml

```
@@ -5,6 +5,11 @@ on:
     types:
       - published
 
+# https://docs.github.com/en/rest/overview/permissions-required-for-github-apps
+permissions:
+ contents: write 
```

```suggestion
 contents: read 
```

Perhaps `read` might be enough as our canary pipeline seems to be working fine with `read`? (I wonder if we need `write` for cache writing but apparently not?